src/components/Team/index.tsx

```diff
@@ -0,0 +1,170 @@
+    name: 'Agustin Longoni',
+    description: 'UX/UI Designer',
+    contact: '@alongoni',
+  },
+  {
+    avatar:
+      'https://s3.us-west-2.amazonaws.com/secure.notion-static.com/b5e7c29c-4240-4ff7-bd7f-f20dad63ff04/photo_2022-09-29_23-43-15.jpg?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Content-Sha256=UNSIGNED-PAYLOAD&X-Amz-Credential=AKIAT73L2G45EIPT3X45%2F20221004%2Fus-west-2%2Fs3%2Faws4_request&X-Amz-Date=20221004T191036Z&X-Amz-Expires=86400&X-Amz-Signature=8843a4c2d3b204cf0b9d820421538687a7eecfd7eeda79f2f9bcb7dbc08bb898&X-Amz-SignedHeaders=host&response-content-disposition=filename%20%3D%22photo_2022-09-29_23-43-15.jpg%22&x-id=GetObject',
+    name: 'Agustin Lombardi',
+    description: 'Community Manager, Funds Raising',
+    contact: '@lombarcrypto',
+    contactType: 'Twitter',
+  },
+  {
+    avatar:
+      'https://s3.us-west-2.amazonaws.com/secure.notion-static.com/8e6fbe90-bb40-4c09-b06a-45041aca0dd5/EUG_7073_Original.jpg?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Content-Sha256=UNSIGNED-PAYLOAD&X-Amz-Credential=AKIAT73L2G45EIPT3X45%2F20221004%2Fus-west-2%2Fs3%2Faws4_request&X-Amz-Date=20221004T191221Z&X-Amz-Expires=86400&X-Amz-Signature=29bef834f0b05661212840e221ee2414de9e023410504e3050cb2764e85648be&X-Amz-SignedHeaders=host&response-content-disposition=filename%20%3D%22EUG_7073_Original.JPG.jpg%22&x-id=GetObject',
+    name: 'Nicolas Magri',
+    description: 'Chief Compliance Officer',
+    contact: '@monito313',
+    contactType: 'Twitter',
+  },
+];
+
+export const Team = () => {
+  return (
+    <Box sx={{ marginTop: 4 }}>
+      <Divider sx={{ borderColor: '#66F5DF' }} />
+
+      <SectionTitle>Team</SectionTitle>
+      <Box
+        sx={{
+          display: 'flex',
+          flexFlow: 'row wrap',
+          columnGap: 3,
+          rowGap: 8,
+          marginBottom: 4,
+        }}
+      >
+        {TEAM_MEMBERS.map((member, i) => {
+          return (
+            <MemberBox key={i}>
+              <Avatar
+                alt={member.name}
+                src={member.avatar}
+                sx={{
+                  width: 92,
+                  height: 92,
+                }}
+                imgProps={{
+                  sx:
+                    i === 6
+                      ? {
+                          height: '200%',
+                          top: '-1px',
+                          position: 'absolute',
+                        } // Agu Lomboardi picture need be diff, hardcoded fix
+                      : {},
+                }}
+              />
+              <MemberName>{member.name}</MemberName>
+              <MemberDescription>{member.description}</MemberDescription>
+              {member.contact && (
+                <Box
```

Comment:
If possible add the src property so they can be clickeable, remember the "target=_blank"

Reply:
Actually it should be easy, you already have the "if"  for twitter or github you can just put the url https://twitter.com/${member.contact}} or https://github.com/${member.contact} and leave the "@" for the component